src/main/java/com/ncc/aif/ValidateAIFCli.java

```diff
@@ -106,7 +111,8 @@
 
     @Option(names = "--abort", description = "Abort validation after [num] SHACL violations (num > 2), or 3 violations if [num] is omitted.",
             paramLabel = "num", arity = "0..1", converter = MaxErrorConverter.class)
-    private int maxValidationErrors = DEFAULT_MAX_VIOLATIONS;
+    private int maxValidationErrors = Integer.MIN_VALUE; // Don't fail-fast by default
```

Comment:
I thought when validationErrorBatch was -1, that turned fail-fast off. Reading this makes me think that fail-fast will now always perform the violation count check. It just won't ever throw the exception.Let me know what I'm missing

Reply:
In `ValidateAIF` itself, `-1` turns off fail-fast, and still does.  In ValidateAIFCli, `-1` was being used to detect whether or not the user was trying to enable fail-fast.  However, I noticed that both `--abort` and `--depth` didn't handle it properly when a parameter of `-1` was specified by the user.  It disabled the feature instead of giving an error saying the parameter was too small.

Instead, I'm setting the variables (`maxValidationErrors` and `depth`) to `Integer.MIN_VALUE` and using that to determine if the user tried to enable the feature.  If they don't specify the flags, then the feature is not enabled.

Reply:
Okey dokey. Marking resolved.